src/lib/rest/rest.cpp

```diff
@@ -413,17 +414,26 @@ static int outFormatCheck(ConnectionInfo* ciP)
 
 /* ****************************************************************************
 *
-* servicePathCheck - 
+* servicePathCheck - check vector of service paths
```

Comment:
This function, as it checks the validity for the URI-parameter should remain here.
The other function with the same name (I don't have any problem at all with that - overloading is a common technique) could be moved to another file, but it doesn't belong here as it isn't used here.
I already put it in the best place I could find ...  [ doesn't mean I'm right - if you find a better place for it, shoot ]